src/storage/storage.h

```diff
@@ -107,18 +108,18 @@ inline const std::vector<CacheOption> CacheOptions = {
     {BlockCacheType::kCacheTypeHCC, "hcc", "kCacheTypeHCC"},
 };
 
-enum class StatType {
+enum class StatType : uint_fast8_t {
   CompactionCount,
   FlushCount,
   KeyspaceHits,
   KeyspaceMisses,
 };
 
-struct DBStats {
-  std::atomic<uint64_t> compaction_count = 0;
-  std::atomic<uint64_t> flush_count = 0;
-  std::atomic<uint64_t> keyspace_hits = 0;
-  std::atomic<uint64_t> keyspace_misses = 0;
+struct alignas(CACHE_LINE_SIZE) DBStats {
```

Comment:
Like rocksdb, an Per core array optimization can also be introduced here. Maybe I'll try it in another patch.

Reply:
Another problem is that, should we change `DBStats` to `unique_ptr<DBStats>` in Storage? @git-hulk @PragmaTwice

Reply:
I feel good about both of them since the lifetime of DBStats is the same as the storage.

Reply:
Emm I think previously it's size is 8*8B=64B, now `sizeof(DBStats)` becomes 8 * 64B. Which would be a large object